docs/examples/prototypes.md

```diff
@@ -1,5 +1,57 @@
 # Prototypes 
 
-## Available Prototypes
+You would like to explore the applications that you can build with Amadeus Self-Service APIs? 
+we have prototypes available in [GitHub](https://github.com/amadeus4dev).
+
+
+Here in this page, we have listed down our prototypes(examples or demos) based on **use case** for your convinence. 
```

Comment:
I think that one was corrected to, but now it looks like before the word convenience.